ajax/libs/unveil2/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "unveil2",
+  "filename": "jquery.unveil2.min.js",
+  "description": "A very lightweight jQuery plugin to lazy load images",
+  "homepage": "https://nabble.github.io/unveil2/",
+  "author": "Joram van den Boezeme",
+  "keywords": [
+    "lazyload",
+    "jquery"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nabble/unveil2.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nabble/unveil2.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
```

Comment:
Let's also add the file jquery.unveil2.js which can be use in web front-end, also it's non-minified file of jquery.unveil2.min.js.
```js
      {
        "basePath": "src",
        "files": [
          "jquery.unveil2*"
        ]
      }
```

The whole auto-update config is 
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/nabble/unveil2.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      },
      {
        "basePath": "src",
        "files": [
          "jquery.unveil2*"
        ]
      }
    ]
  }
```

Thank you.

Reply:
added and comment updated!